doc/maintaining/installing/install-from-source.rst

```diff
@@ -277,6 +277,10 @@ Follow the instructions in :doc:`/maintaining/datastore` to create the required
 databases and users, set the right permissions and set the appropriate values
 in your CKAN config file.
 
+After setting up the DataStore, configure DataPusher or XLoader to add data to DataStore.
+To install the DataPusher refer this docs: https://github.com/ckan/datapusher or refer this
+docs: https://github.com/ckan/ckanext-xloader to install the XLoader.
+
 ---------------
```

Comment:
Once you have set up the DataStore, you may then wish to configure either the DataPusher _or_ XLoader extensions to add data to the DataStore. To install DataPusher refer to this link: https://github.com/ckan/datapusher or refer to this link: https://github.com/ckan/ckanext-xloader to install XLoader.